frontend/src/pages/AppList/AppListPage.tsx

```diff
@@ -74,6 +74,11 @@ class AppListPageComponent extends FilterComponent.Component<AppListPageProps, A
     this.promises.cancelAll();
   }
 
+  onSort = (): void => {
+    // force list update on sorting
+    this.setState({});
+  };
```

Comment:
I guess this is this a bug fix?

Reply:
Yes. For list sorting, we modified the URL parameter in the pathname. React Router v5 reloads the component if the pathname is modified, but React Router v6 does not. Therefore, I have to force the reload of the list component with a setState to see the sorting.